frontend/src/metabase/query_builder/components/view/ViewSection.jsx

```diff
@@ -4,8 +4,15 @@ import { Flex } from "grid-styled";
 
 import Subhead from "metabase/components/Subhead";
 
-const ViewSection = ({ bottom, trim, style, className, ...props }) => (
-  <Flex align="center" px={3} style={style} {...props} className={className} />
+const ViewSection = ({
+  align = "center",
+  bottom,
+  trim,
+  style,
+  className,
+  ...props
+}) => (
+  <Flex align={align} px={3} style={style} {...props} className={className} />
```

Comment:
I think we'll want to adjust the `px` value here so that it's a little tighter on mobile. I'd go with `px={[1,3]}`